### PR TITLE
Air 2059: Load asset page server-side for public assets

### PR DESCRIPTION
**Pull request by codiak**

### CODE REVIEW

File: src/app/_services/metadata.service.ts [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+})
+export class MetadataService {
+
+  constructor(
+      private _auth: AuthService,
+      private _http: HttpClient
+  ) { }
+
+
+    public buildAsset(assetId: string, { groupId = '', legacyFlag = true, openlib = false, encrypted = false }={}): Observable<Asset> {
```
Comment (bFraley) [inline src/app/_services/metadata.service.ts]: Do we want to set legacyFlag to true here ?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -22,16 +26,16 @@ import { DomUtilityService } from 'app/shared';
   template: `
     <ang-sky-banner *ngIf="showSkyBanner" [textValue]="skyBannerCopy" (closeBanner)="showSkyBanner = false"></ang-sky-banner>
     <div>
-      <div id="skip" tabindex="-1" aria-activedescendant="button">
+      <div id="skip" tabindex="-1">
         <button id="button" (click)="findMainContent()" (keyup.enter)="findMainContent()" tabindex="1" class="sr-only sr-only-focusable"> Skip to main content </button>
       </div>
       <nav-bar tabindex="-1"></nav-bar>
-    </div>
-    <main tabindex="-1">
```
Comment (bFraley) [inline src/app/app.component.ts]: Will this break tabbing order / past accessibility work ? I 'm sure we can review that when looking at merging into stage branch

File: src/app/app.component.ts [hunk 3]
```diff
@@ -53,20 +57,20 @@ export class AppComponent {
...
-      translate.setDefaultLang(langStr);
-      // the lang to use, if the lang isn't available, it will use the current loader to get them
-      translate.use(langStr);
-
+      // Use the translate loader to pull translations client-side
+      this.translate.use(langStr);
+      this.translate.setDefaultLang(langStr);
       // Start GA trackiong
       angulartics2GoogleAnalytics.startTracking()
+    }  else {
```
Comment (bFraley) [inline src/app/app.component.ts]: This is great!

**Comment by bFraley** [comment]
@codiak Great work! Let's merge!

File: src/app/_services/metadata.service.ts (hunk 1 above)
Comment (codiak) [reply src/app/_services/metadata.service.ts]: Good catch...

**Pull request merged**